Source/Hydrate.spoon/init.lua

```diff
@@ -0,0 +1,124 @@
+--- Friendly Reminder to remind you to hydrate.
+---
+--- This spoon will display a reminder to hydrate every X minutes.
+--- You can change the interval in the `reminder_interval_minutes` variable.
+--- You can also customize the day of the week to remind you by setting the `reminder_day_of_week` variable.
+---
+--- Usage:
+--- ```
+---     hs.loadSpoon("Hydrate")
+---     spoon.Hydrate.reminder_interval_minutes = 60
+---     spoon.Hydrate.reminder_days = {"Monday", "Wednesday", "Friday"}
+---     spoon.Hydrate:start()
+--- ```
+---
+
+local obj={}
+obj.__index = obj
+
+
+local Notifier = {}
+Notifier.__index = Notifier
+
+-- Metadata
+obj.name = "Hydrate"
+obj.version = "0.1"
+obj.author = "Luiz Marques <luizfelipe.unesp@gmail.com>"
+obj.license = "MIT - https://opensource.org/licenses/MIT"
+
+--- Hydrate.reminder_interval_minutes
+--- Variable
+--- Interval in minutes to show the reminder. Default is 30 minutes.
+obj.reminder_interval_minutes = 30
+
+--- Hydrate.reminder_days
+--- Variable
+--- Days of the week to show the reminder. Default is every day.
+obj.reminder_days = {
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday"
+}
+
+--- Hydrate.Notifier
```

Comment:
This API is going to be tricky to express usefully with the way our docstrings work, because `Notifier` isn't defined anywhere as a module, and in fact it's not a separate module (and if it were, it would get its own page in the docs).

I wonder if it would be possible to restructure things a bit so the API is all in the `Hydrate` namespace?

Reply:
Thanks for the advice @cmsj . This is literally my first interaction w/ hammerspoon (and lua). I will take a look in other projects and the docs and come up w/ some solution :)

Reply:
In theory, if you rebase your branch off our `master` branch, you should get docstrings errors as comments now. I'm pretty sure I've finally figured out a way to make that work :)

Reply:
just rebased, lets see :)

Reply:
I rebased asap I saw your comment @cmsj but I guess the changes weren't in master yet, due to CI delays. I just did it again now. Could you allow CI to run again pls?